components/blocks/iconHeader.module.css

```diff
@@ -13,27 +13,27 @@
 
 /* Color modifiers */
 .VioletBackground {
-  @apply bg-indigo-70;
+  @apply bg-[#B02AAC];
 }
 
 .OrangeBackground {
   @apply bg-orange-70;
 }
 
 .LightBlueBackground {
-  @apply bg-lightBlue-70;
+  @apply bg-[#2EAADC];
```

Comment:
I am wondering if we should move these custom values to tailwind config with easier naming. This way we will be able to reuse them everywhere